ts/parts/Tooltip.ts

```diff
@@ -1049,19 +1049,7 @@ H.Tooltip.prototype = {
             !((point as any).series &&
             (point as any).series.noSharedTooltip)
         ) {
-            // Set inactive state for all points
-            activeSeries = chart.pointer.getActiveSeries(point as any);
-
-            chart.series.forEach(function (
-                inactiveSeries: Highcharts.Series
-            ): void {
-                if (
-                    inactiveSeries.options.inactiveOtherPoints ||
-                    activeSeries.indexOf(inactiveSeries) === -1
-                ) {
-                    inactiveSeries.setState('inactive', true);
-                }
-            });
+            chart.pointer.setInactiveSeries(point as Array<Highcharts.Point>);
```

Comment:
The cast should not be necessary if the outer conditional is type complete. If it is incomplete please cast to any, so that we can pick it up later.

Reply:
Yup, I was really curious why casting to `any` when I explicite know the type?

Reply:
Cast to any will be later in the migration process forbidden and is kind of our todo type, so that the compiler gets quiet.

Reply:
Okay! Let me change then, thanks :)